app/code/Magento/Captcha/etc/frontend/di.xml

```diff
@@ -37,4 +37,18 @@
     <type name="Magento\Sales\Api\OrderManagementInterface">
         <plugin name="reset_payment_attempts_after_order_is_placed_plugin" type="Magento\Captcha\Plugin\ResetPaymentAttemptsAfterOrderIsPlacedPlugin"/>
     </type>
+    <type name="Magento\Checkout\Block\Onepage">
+        <arguments>
+            <argument name="layoutProcessors" xsi:type="array">
+                <item name="checkout_processor" xsi:type="object">Magento\Captcha\Block\CheckoutLayoutProcessor</item>
+            </argument>
+        </arguments>
+    </type>
+    <type name="Magento\Customer\Block\Account\AuthenticationPopup">
+        <arguments>
+            <argument name="layoutProcessors" xsi:type="array">
+                <item name="captcha_processor" xsi:type="object">Magento\Captcha\Block\Checkout\LayoutProcessor</item>
```

Comment:
The current name of the class is quite confusing:
- it's almost the same as  `Magento\Captcha\Block\CheckoutLayoutProcessor`, the differnce is the only one bachslash
- It's not linked anyhow to customer and to authentication popup
So I'm suggesting renaming the class/namespace + update it in di.xml to have something like this
```suggestion
                <item name="captcha_processor" xsi:type="object">Magento\Captcha\Block\Customer\AuthenticationPopup\LayoutProcessor</item>
```